File: tests/fixtures/java/maven_pkg/src/main/java/com/example/animals/Sound.java
package com.example.animals;

public final class Sound {
    private final String text;

    public Sound(String text) {
        this.text = text;
    }

    public String text() {
        return text;
    }

    public Sound amplify() {
        return new Sound(text + "!");
    }
}

File: tests/fixtures/cpp/basic_pkg/src/sound.cpp
#include "sound.h"

#include <utility>

namespace acme {

Sound::Sound(std::string text) : text_(std::move(text)) {}

std::string Sound::text() const { return text_; }

Sound Sound::amplify() const { return Sound(text_ + "!"); }

}  // namespace acme
